src/index.js

```diff
@@ -0,0 +1,276 @@
+      throw new TypeError(
+        `Cannot apply transform '${transform}', is not a supported BigNumber.js method`,
+      );
+    } else {
+      return bigNumber[transform]();
+    }
+  }
+}
+
+Cents.prototype.lt = Cents.prototype.lessThan;
+Cents.prototype.lte = Cents.prototype.lessThanOrEqual;
+Cents.prototype.gt = Cents.prototype.greaterThan;
+Cents.prototype.gte = Cents.prototype.greaterThanOrEqual;
+
+Cents.isValid = function (maybeCents) {
+  let centsInstance;
+  try {
+    centsInstance = new Cents(maybeCents);
+  } catch {
+    centsInstance = new Error('invalid');
+  }
+  return centsInstance instanceof Cents;
+};
+
+Cents.isValidDollars = function (maybeDollars) {
+  let threw = false;
+  try {
+    Cents.fromDollars(maybeDollars);
+  } catch {
+    threw = true;
+  }
+  return !threw;
+};
+
+Cents.fromDollars = (
+  dollars, // dollars should be a Number like xx.yy
+) => new Cents(new BigNumber(dollars).times(100));
+
+Cents.round = function (maybeInt) {
+  if (!(maybeInt >= 0)) {
+    throw new Error(`${maybeInt} must be positive to round to cents`);
+  }
+  return new Cents(new BigNumber(maybeInt).integerValue(BigNumber.ROUND_HALF_UP));
+};
+
+// This method supports static method calls of the form:
+//   Cents.staticMethod(...)
+//
+// The ... must include at least one argument.
+//
+// Case 1: ... => single argument array
+//   There must be no other arguments and each value in the array must "pass" the
+//   validator (boolean) predicate function. Returns the array.
+// Case 2: ... => multiple argument values
+//   Each value must "pass" the validator predicate function. Returns an array of the values.
+const arrayifySplat = function (splat, validator) {
+  if (!(splat.length > 0)) {
+    throw new Error('Expect at least one argument');
+  }
+
+  if (Array.isArray(splat[0])) {
+    if (splat.length !== 1) {
+      throw new Error('Expect a single array argument');
+    }
+    splat = splat[0];
+  }
+
+  if (validator != null) {
+    splat.forEach(function (val) {
+      if (!validator(val)) {
+        throw new Error(`Unexpected value ${val}`);
+      }
+    });
+  }
+
+  return splat;
+};
+
+Cents.min = function (...cents) {
+  cents = arrayifySplat(cents, Cents.isValid);
+  cents = cents.map((cent) => new Cents(cent).toNumber());
+  const min = Math.min(...cents);
+  return new Cents(min);
+};
+
+Cents.max = function (...cents) {
+  cents = arrayifySplat(cents, Cents.isValid);
+  cents = cents.map((cent) => new Cents(cent).toNumber());
+  const max = Math.max(...cents);
+  return new Cents(max);
+};
+
+Cents.sum = function (...cents) {
+  cents = arrayifySplat(cents, Cents.isValid);
+  return cents.reduce((memo, val) => memo.plus(val), new Cents(0));
+};
+
+Cents.sumDollars = function (...dollars) {
+  dollars = arrayifySplat(dollars, Cents.isValidDollars);
+  return dollars.reduce((memo, val) => memo.plus(Cents.fromDollars(val)), new Cents(0));
+};
+
+// eslint-disable-next-line import/no-commonjs
+module.exports = Cents;
```

Comment:
We can make this a major version. We use ES modules everywhere now.
```suggestion
export default Cents;
```

Reply:
Makes sense, I leaved it due I was concern about backward support where ever this library is being used. But changing it now.